wcfsetup/install/files/lib/util/HTTPRequest.class.php

```diff
@@ -97,9 +98,10 @@ public function __construct($url, array $options = [], $postParameters = [], arr
 
         // set default headers
         $language = WCF::getLanguage();
+        $domain = ApplicationHandler::getInstance()->getApplicationByID(1)->domainName;
         $this->addHeader(
             'user-agent',
-            "HTTP.PHP (HTTPRequest.class.php; WoltLab Suite/" . WCF_VERSION . "; " . ($language ? $language->languageCode : 'en') . ")"
+            "HTTP.PHP (HTTPRequest.class.php; WoltLab Suite/" . WCF_VERSION . "; " . ($language ? $language->languageCode : 'en') . "; " . $domain . ")"
```

Comment:
The User-Agent format in `HTTPRequest` is intentionally unchanged with the introduction of Guzzle for compat reasons, because I know that some services are parsing the User-Agent header for dumb reasons.